clang/test/Sema/tautological-pointer-comparison.c

```diff
@@ -1,40 +1,72 @@
 // RUN: %clang_cc1 -fsyntax-only -verify %s
+// RUN: %clang_cc1 -fsyntax-only -DFWRAPV -fwrapv -verify %s
```

Comment:
I'm not very familiar with `-verify` tests, but may there is some way to avoid these ifdefs using prefixes? Something like this maybe?
```suggestion
// RUN: %clang_cc1 -fsyntax-only -DFWRAPV -fwrapv -verify=fwrapv %s

// fwrapv-no-diagnostics
```

Reply:
Ah yes, that is much cleaner. Done in 8504a0d758c06fa9f3b95117fdb05021d75afe8d.